protos/common.proto

```diff
@@ -1,46 +1,52 @@
-syntax = "proto3";
```

Comment:
Looks like you've added a CR to all line endings. Was this on purpose or accidentally (which is probably easy on Windows)? Might be worth adding `core.autocrlf true` a local Git config, wdyt?

Reply:
Completely accidental. Added the config